### ISSUE TITLE
Unique_id is missing

### ISSUE THREAD

**Issue by GSzabados** [simontb/mojio-home-assistant#20]
Hi @simontb & @themoosman ,

Would be possible to add a unique_id to the entities created by the integration?

The VIN number or the vehicle.mojio_id could be used as unique_id. The first would be fixed to a car, if the device reads it, the second would be fixed by mojio (I guess...). Whichever would be easier to implement...

The unique_id would be necessary to expose entities to Assist or any other voice assistant, since 2023.5

Thanks in advance!

Best Regards,

Gábor

PS.: And if you are there would you please merge PR #19

**Comment by simontb** [comment]
Yes, I guess this can be done. I don't know the unique_id. Guess this should be unique among all integrations. Then I wouldn't use the VIN, because another integration might use it (like your car manufacturers API). What's the format of the Mojio ID? Is it a UUID or a format with potential random overlaps as well? Maybe we have to add a prefix like 'mojio-'.

**Comment by themoosman** [comment]
I think we just use the mojio id as it's a UUID, and it's already part of the vehicle.  https://github.com/simontb/mojio-py/blob/master/mojio_sdk/vehicle.py#L25

**Comment by GSzabados** [comment]
Regarding unique_id:

https://www.home-assistant.io/faq/unique_id

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-ids
